polynote-frontend/polynote/ui/input/hotkeys.ts

```diff
@@ -31,6 +35,10 @@ export function getHotkeys() {
     return hotkeys;
 }
 
+export function keycodeToKeybinding(code: number) {
```

Comment:
Looks like this is only used in a log message that is slated for removal in cell.ts?